src/components/app/Home.js

```diff
@@ -6,9 +6,9 @@
 
 import * as React from 'react';
 
-import { AppHeader } from './AppHeader';
+import { AppHeader } from 'firefox-profiler/components/app/AppHeader';
```

Comment:
For example this import is happening from the same directory. We should keep the old change like `./`

Reply:
@canova thank you for the feedback, let me change that and push the change